html-templates/progress/section-term-reports/reports.email.tpl

```diff
@@ -0,0 +1,97 @@
+{$from = $.User->EmailRecipient}
+
+{capture assign=subject}{strip}
+
+    {if count($students) > 1}
+        Term reports for {count($students)} students
+    {else}
+        Term report for {$students[0]->FullName}
+    {/if}
+
+    {if count($terms) == 1}
+        , {$terms[0]->Title}
+    {/if}
+
+{/strip}{/capture}
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>{$subject|escape}</title>
+
+        <style type="text/css">
+            .term a:visited { color: #a35500 !important; }
+            .term a:hover { color: #6a3700 !important; }
+            .term a:active { color: #6a3700 !important; }
```

Comment:
@nbey these `.term` and `.interim` classes don't appear to be used anywhere in the HTML. We should probably just remove the class name prefix so these styles apply to all links in the email